requirements.txt

```diff
@@ -6,8 +6,8 @@
 #
 ansible-modules-hashivault==3.9.6 \
     --hash=sha256:14c9636931309cadabc62298f71758658bc2331cd16a81dce99d359f73e507af
-ansible==2.6.0 \
-    --hash=sha256:3550bf16bf2c34cc88bf3870e91b836e44c1c288d148968c4d855dae2560faf5
+ansible==2.6.1 \
```

Comment:
We should keep this in sync with the `requirements.txt` used by `make shell` somehow.

Reply:
So we have ansible here to import it and parse inventory (for testing that prometheus endpoint gather all endpoint it should gather). For installation, this is ansible of `make shell` which is used.
As the internal API of ansible might not be considered as "stable", we might want to keep an "old" version here.
But in the mean time I don't want us to be stuck at an old version of ansible for the run of metal-k8s itself.